pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -536,6 +543,43 @@ self: super: {
     dependencies = with self; [ skim ];
   });
 
+  sniprun =
```

Comment:
I didn't add `michaelb/sniprun` to `vim-plugin-names` because it should NOT.

It requires to update the sha256 hashes of sniprun to update other plugins, which makes maintaining other plugins so complex.